### DISCUSSION TITLE
How to group REST API with openapi oas 3 like SpringBoot GroupedOpenApi

### DISCUSSION THREAD

**Discussion by hittlle** [micronaut-projects/micronaut-core#7840] [Q&A] [answered]
Could anyone tell me how to group REST API OAS 3 like SpringBoot GroupOpenApi?

**Comment by altro3** [comment, marked as answer]
I understand that it is very late, but I will still write an answer so that it is clear to everyone.

Versioning support has already been implemented and described in the documentation https://micronaut-projects.github.io/micronaut-openapi/latest/guide/index.html#versions

Also, support for different API versions is implemented using `@OpenAPIGroup` annotaion. Microsoft OpenAPI has excellent support for grouping endpoints and splitting them into several specifications. Much cooler and more convenient than anywhere else

If you have questions, suggestions or problems, please create tickets here: https://github.com/micronaut-projects/micronaut-openapi/issues . Discussions are not reviewed regularly and rarely
